modules/analysis/rosbag-image-pipeline/modulestack.yaml

```diff
@@ -25,25 +28,29 @@ Resources:
               - "s3:Get*"
               - "s3:List*"
             Effect: Allow
-            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+            Resource: 
+              - !Sub 
+                - "arn:aws:s3:::${ProjectName}*"
+                - ProjectName: !Ref ProjectName
+              - "arn:aws:s3:::cdk*"
           - Effect: Allow
             Action:
               - "ecr:Describe*"
               - "ecr:Get*"
               - "ecr:List*"
               - "ecr:BatchGetImage"
             Resource:
-              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/addf-${DeploymentName}-${ModuleName}*"
+              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/${ProjectName}-${DeploymentName}-${ModuleName}*"
           - Effect: Allow
             Action:
               - ecr:*
             Resource:
-              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/addf-${DeploymentName}-${ModuleName}*"
+              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/${ProjectName}-${DeploymentName}-${ModuleName}*"
           - Action:
               - "iam:CreateServiceLinkedRole"
```

Comment:
Do you need servicelinkedrole permissions here?

Reply:
This is from Kevin Soucy's initial commit 2 years ago. Should we test this with removal now or in a separate PR? @dgraeber @srinivasreddych

Reply:
Not sure why it is even there to be honest

Reply:
I vote merging this as it fits the scope and then opening a separate issue, to refine the module's permissions

Reply:
https://github.com/awslabs/autonomous-driving-data-framework/issues/503